src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4025,6 +4073,9 @@ def create_functionapp(cmd, resource_group_name, name, storage_account, plan=Non
     poller = client.web_apps.begin_create_or_update(resource_group_name, name, functionapp_def)
     functionapp = LongRunningOperation(cmd.cli_ctx)(poller)
 
+    if environment is not None and (workload_profile_name is not None or cpu is not None or memory is not None):
+        functionapp = client.web_apps.get(resource_group_name, name)
+
```

Comment:
Do we still need this?

Reply:
Yes

Reply:
Wasn't this just a temporary workaround until the control plane changes returned this in the PUT call?